chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/CertifiedProductController.java

```diff
@@ -11,6 +11,7 @@
 import java.util.Date;
 import java.util.HashSet;
 import java.util.List;
+import java.util.Map;
```

Comment:
I don't think this import is used

Reply:
That's correct. I should have removed it (and will update accordingly). Normally in the past I go ahead and run 'organize imports' automatically or as part of a team-based save-cleanup automation. I wasn't sure if I should run it here since others might not be doing that and it might cause the code to look very different. If we all did it, or it was automated on save for us all, we wouldn't ever have such issues or concerns.

Reply:
I tend to do the Eclipse `Ctrl-Shift-O` command as I'm getting ready for PR. I _think_ that's the "organize imports" command ;)

If you can figure out how to automate that on save I'd be all about incorporating that into our processes. For that matter, any other automated way of cleaning up our code is welcomed and encouraged (with the caveat about too many "changed lines" makes it hard to parse a PR). That said, organizing the imports shouldn't be problematic

Reply:
Yeah that's the hotkey. I would suggest that the team add organize imports (it's a checkbox) to the save action in Preferences->Editor->Save Actions. Another great one to add for this project's requirements specifically is 'Remove trailing white spaces on all lines.' For that one you would want to go in and edit out everything there by default except that. And in the future maybe there are other items we can automate that are universally understood as a team requirement so they won't cause mass changes. I am certainly all about the computer spending time caring about blank spaces vs me.